README.md

```diff
@@ -83,10 +84,51 @@ You can do that in your `config/database.php` file, where you define your databa
 
 ## Usage
 
-Point and click, baby. Point and click.
+This should be a point-and-click interface where you can create and download backups at any time.
 
 Try at **your-project-domain/admin/backup**
 
+## Upgrading from 3.x to 4.x
+
+Change your required version to `"backpack/backupmanager": "^4.0"` and run `composer update backpack/backupmanager`.
+
+We removed the overrides of spatie config from our package publishing process, so going foward you can do the regular spatie backup configuration in `config/backup.php` and the customized backpack configuration in `config/backpack/backupmanager`.
+**1)** publish the new config file `php artisan vendor:publish --provider="Backpack\BackupManager\BackupManagerServiceProvider" --tag="backup-config"`. This will generate your the `config/backpack/backupmanager.php` file. By default backpack uses `--disable-notifications` flag, remove it if you are using notifications.
```

Comment:
```suggestion
**1)** publish the new config file `php artisan vendor:publish --provider="Backpack\BackupManager\BackupManagerServiceProvider" --tag="backup-config"`. This will generate the `config/backpack/backupmanager.php` file. By default backpack uses `--disable-notifications` flag, remove it if you are using notifications.
```